src/test/kotlin/no/nav/syfo/api/endpoints/ArbeidsuforhetEndpointsSpek.kt

```diff
@@ -90,6 +103,89 @@ object ArbeidsuforhetEndpointsSpek : Spek({
                             pVarselPdf?.pdf?.get(1) shouldBeEqualTo PDF_FORHANDSVARSEL[1]
                         }
                     }
+                    it("Successfully gets an existsing forhandsvarsel") {
+                        runBlocking {
+                            forhandsvarselService.createForhandsvarsel(
+                                personident = PersonIdent(personIdent),
+                                veilederident = VEILEDER_IDENT,
+                                begrunnelse = begrunnelse,
+                                document = document,
+                                callId = UUID.randomUUID().toString(),
+                            )
```

Comment:
Ville kanskje heller brukt en spørring direkte mot dben for å legge inn noe her? Så frikobler man api-testen fra servicen i større grad.

Reply:
Ja, kan se om det er like enkelt.

Reply:
Pratet litt mer om det nå - lar det være sånn som dette i første omgang.